src/core/client.ts

```diff
@@ -469,24 +460,22 @@ export default class SlashAuthClient {
    *
    * @param options
    */
-  public async getTokenSilently(
-    options: GetTokenSilentlyOptions & { detailedResponse: true }
-  ): Promise<GetTokenSilentlyVerboseResponse>;
+  public async getTokens(
+    options: GetTokensOptions & { detailedResponse: true }
+  ): Promise<GetTokensVerboseResponse>;
 
   /**
    * Fetches a new access token and returns it.
    *
    * @param options
    */
-  public async getTokenSilently(
-    options?: GetTokenSilentlyOptions
-  ): Promise<string | null>;
+  public async getTokens(options?: GetTokensOptions): Promise<string | null>;
 
   /**
    * Fetches a new access token, and either returns just the access token (the default) or the response from the /oauth/token endpoint, depending on the `detailedResponse` option.
    *
    * ```js
-   * const token = await slashauth.getTokenSilently(options);
+   * const token = await slashauth.getTokens(options);
```

Comment:
does this return one token or multiple tokens? the left-side seems to be one token

Reply:
It can return multiple (if the parameter of verbose is passed)